it/spring/boot2-tomcat8/build.gradle

```diff
@@ -1,5 +1,5 @@
 final def SPRING_BOOT_VERSION = '2.1.15.RELEASE'
```

Comment:
Don't we need to upgrade to 2.1.16? 
https://github.com/spring-projects/spring-boot/releases/tag/v2.1.16.RELEASE

Reply:
We do. :sweat_smile: